README.md

```diff
@@ -31,7 +26,7 @@ Note the `type="module"`, that's important.
 
 ```html
 <head>
-  <script type="module" src="https://unpkg.com/hls-video-element@0.0"></script>
+  <script type="module" src="https://unpkg.com/hls-video-element@0"></script>
```

Comment:
When we're in the v0 realm we should pin people to specific minor versions, so we can have breaking changes in the minor version with less risk of breaking people. That's at least what we've been doing with media chrome.

Reply:
good idea, will change it back